internal/testplanet/planet.go

```diff
@@ -517,12 +518,13 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 				PasswordCost:    console.TestPasswordCost,
 				AuthTokenSecret: "my-suppa-secret-key",
 			},
+			Vouchers: satVouchers.Config{
```

Comment:
Remember to let devops know about these config changes in the config-changes slack channel